components/kyma-environment-broker/internal/process/queue.go

```diff
@@ -68,7 +69,7 @@ func worker(queue workqueue.RateLimitingInterface, process func(key string) (tim
 				log = log.WithField("operationID", id)
 				defer func() {
 					if err := recover(); err != nil {
-						log.Errorf("panic error from process: %v", err)
+						log.Errorf("panic error from process: %v\n%s", err, debug.Stack())
```

Comment:
After looking at the result log, this should be modified like "panic error from process: %v. Stacktrace: %s", and also in parallel orchestration strategy. I think this should be merged as it's very useful to capture bugs in operation processing.